app/models/ems_cluster.rb

```diff
@@ -51,10 +51,7 @@ class EmsCluster < ApplicationRecord
   self.default_relationship_type = "ems_metadata"
 
   include AggregationMixin
-  # Since we've overridden the implementation of methods from AggregationMixin,
-  # we must also override the :uses portion of the virtual columns.
-  override_aggregation_mixin_virtual_columns_uses(:all_hosts, :hosts)
-  override_aggregation_mixin_virtual_columns_uses(:all_vms_and_templates, :vms_and_templates)
+  aggregation_mixin_virtual_columns_use # use default uses
```

Comment:
I'm fine with this PR overall, but is there a way to get rid of these calls entirely for the defaults?